index.html

```diff
@@ -2390,6 +2392,11 @@
           controllerMesh.quaternion.fromArray(gamepad.pose.orientation);
           controllerMesh.updateMatrixWorld(true);
           controllerMesh.visible = true;
+          if(orbitControls || currentSession === 'none'){
```

Comment:
maybe this should just be ` if(currentSession === 'none'){`

Reply:
I think we were already tracking the XR device/session. So there is not need for `currentSession`.

Reply:
I see there  is a private `session`, the session is assigned to the display

https://github.com/exokitxr/studio/blob/d7016fab67fe94614759c7d21569ae656c70b286/index.html#L3223-L3226

so the goal would to check for `fakeXrDisplay.session / xrDisplay.session` ?

Reply:
That's probably just bad code. We shouldn't be attaching `session` to `fakeXrDisplay` if it has nothing to do with the display. This is just legacy from previous versions of `realitytabs.html`/WebXR API.

I think it would make more sense to make `session` a global. But for the purpose of toggling controller rays we don't need to store any globals at all. We just need to identify the points at which that needs to change (this is probably one of them) and do the toggle there. Nothing needs to be stored, other than the controller visibility state.

Reply:
426bfc7 , this will need to be updated with #20 though

Reply:
Looks good 👍.